api/models.cs

```diff
@@ -1,11 +1,11 @@
 using System.Text.Json;
+using StackExchange.Redis;
```

Comment:
File name should start from upper-case letter

Reply:
done